README.md

```diff
@@ -109,6 +110,13 @@ Perform a simple single warmup request (per worker) before starting
 execution. Note that this will be reflected in the counters that
 Nighthawk writes to the output. Default is false.
 
+--request-source-plugin-config <string>
+Request Source plugin configuration in json or compact yaml. Mutually
+exclusive with --request-source. Example (json):
```

Comment:
I believe it's also mutually exclusive with --request-options.  If we are going to document this, we should do so completely/consistently.  Should we also be updating the documentation of the old fields to indicate their mutual exclusivity?

Reply:
That would make sense, but --request-options doesn't actually seem to be a command line options field. I am updating --request-source to indicate their mutual exclusivity though. Currently, the only way we can get request-options seems to be via StartRequest being passed to HandleExecutionRequest.